lumino-explorer-api/src/main/java/org/rif/lumino/explorer/models/documents/LuminoHub.java

```diff
@@ -11,19 +14,37 @@
 
     private String url;
 
-    private Integer connectionCount;
+    private Set<String> connectedClients;
+
+    private long connectionCount;
+
+    private long maxConnectionsAllowed;
 
     public LuminoHub() {
+        this.connectedClients = new HashSet<>();
+        this.connectionCount = 0;
+        this.maxConnectionsAllowed = -1;
     }
 
     public LuminoHub(String url) {
         this.url = url;
+        this.connectedClients = new HashSet<>();
         this.connectionCount = 0;
+        this.maxConnectionsAllowed = -1;
     }
 
-    public LuminoHub(String url, int connectionCount) {
+    public LuminoHub(String url, HashSet<String> connectedClients) {
         this.url = url;
-        this.connectionCount = connectionCount;
+        this.connectedClients = connectedClients;
+        this.connectionCount = connectedClients.size();
+        this.maxConnectionsAllowed = -1;
+    }
+
+    public LuminoHub(String url, long maxConnectionsAllowed) {
+        this.url = url;
```

Comment:
i think we can force that the hubs url must end on /api/v1

maybe the suffix can be a property of the explorer

this way we can validate correct urls and also we dont need to add the sufix on rconnect

Reply:
We can do that if you want, i already added the suffix to rconnect because i thought that the hub url was only the endpoint url (like protocol://host:port) not with the suffix already set, but it's a good suggestion we can do it

Reply:
I will update this on the properties and we assume that the user will set the url properly when adds a new hub to database, in rconnect we assume that /api/v1 will be there, i will do this since i don't like to put strings like /api/v1 in the code directly since tomorrow maybe we change that suffix so i think is better this way